### DISCUSSION TITLE
TEA Protocol

### DISCUSSION THREAD

**Discussion by primero111** [pkgxdev/pkgx#913] [Help]
With $16.9 million in investments from leading entities like Binance Labs, XBTO Humla Ventures, Lattice Capital, and DARMA Capital, Tea Protocol introduces the $TEA token. 

I think TEA will make it. What do you think?

Here's there site if you want to check them out: https://tea.xyz

**Comment by eternize** [comment]
Now that Binance has invested, I think it's going to be great.

**Comment by Claire3653** [comment]
I'm super bullish on TEA. Incentivizing maintenance of
OSS codebase sounds intersting

**Comment by jpekx** [comment]
Bullish on that project too

**Comment by dorikunz** [comment]
I think so too

**Comment by jperica** [comment]
I belive $TEA can do very well. Airdrop can be lucrative as that of ARBITRUM
